internal/pkg/service/buffer/store/export.go

```diff
@@ -108,23 +106,11 @@ func (s *Store) updateExportOp(ctx context.Context, oldValue, newValue model.Exp
 
 	if newValue.OpenedFile.FileID != oldValue.OpenedFile.FileID {
 		now := newValue.OpenedFile.OpenedAt()
-
-		// Close opened file
-		closeFileOp, err := s.setFileStateOp(ctx, now, &oldValue.OpenedFile, filestate.Closing)
-		if err != nil {
-			return nil, err
-		}
-
-		// Close opened slice
-		closeSliceOp, err := s.setSliceStateOp(ctx, now, &oldValue.OpenedSlice, slicestate.Closing)
+		swapFile, err := s.swapFileOp(ctx, now, &oldValue.OpenedFile, &oldValue.OpenedSlice, newValue.OpenedFile, newValue.OpenedSlice)
```

Comment:
Vytvoril som `swap` operacie, ktore jednu slice/file zatvoria a novu ulozia.
Bude sa to pouzivat pri zatvarani suboru, pri importe, tak aby to nebolo duplicitne.

Dozeraz sa to pouzivalo iba pri zmene `mapping-u`.

Reply:
![image](https://user-images.githubusercontent.com/19371734/213557336-7b7be98a-c04c-461d-883e-5575f126ba40.png)